glide.yaml

```diff
@@ -4,6 +4,7 @@ import:
 - package: github.com/mdlayher/arp
 - package: github.com/mdlayher/ethernet
 - package: github.com/mdlayher/ndp
+  version: ff6d60d8d99386184158616d7302471581335768
```

Comment:
Do you need to explicitly lock the version? I tend to prefer leaving it unlocked so that `glide up` automatically pulls the latest, because I will inevitably forget to maintain version pins :(

Reply:
I'm no good at glide. I just did this because I couldn't figure out how to get it to update just my package otherwise, heh.

Reply:
If you just run `glide up`, it should update everything to the latest version, respecting other constraints.

Then again, version management is NP-hard, so it's possible that my attempts to make k8s and gobgp cooperate has borked something else...